src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Bulldozer started its engine and began to work.");
```

Comment:
The message should indicate what type of 'Machine' is working right now. It's better to specify that it's a 'Bulldozer' to make the message more informative.